ExtraTests.cmake

```diff
@@ -69,6 +69,11 @@ opm_add_test( rst_msw
               LIBRARIES ${TEST_LIBS}
               TEST_ARGS tests/MSW.DATA tests/MSW_RESTART.DATA )
 
+add_test( NAME rst_deck_test
+          COMMAND ${PROJECT_SOURCE_DIR}/tests/rst_test_driver.sh ${CMAKE_BINARY_DIR}/bin/rst_deck ${CMAKE_BINARY_DIR}/bin/opmhash
```

Comment:
PROJECT_BINARY_DIR

Reply:
test execution is separate from building, and you cannot add dependencies to the test target in the build system. the best you can do is to make sure it works with our custom 'check' target (which builds, and then executes tests). this is done automatically for anything that is built through EXAMPLE_SOURCE_FILES, so there is nothing more to be done.

Reply:
however, we would want the path of rst_deck to be exposed to downstreams where we will use it. you do this like we do for OPM_PACK on COMPARE_ECL - see https://github.com/OPM/opm-common/blob/master/CMakeLists.txt#L43 and the block below